apps/vx-scan/frontend/src/api/api.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable vx/gts-no-import-export-type */
+import type { Api } from '@votingworks/vx-scan-backend';
```

Comment:
```suggestion
import { type Api } from '@votingworks/vx-scan-backend';
```

Reply:
This still seems to break stylelint. I will follow up and try to upgrade stylelint to fix, but in a separate PR

Reply:
https://github.com/votingworks/vxsuite/issues/2858